go/src/koding/klientctl/endpoint/auth/auth.go

```diff
@@ -44,10 +44,11 @@ type LoginOptions struct {
 	Token    string
 	Username string
 	Password string
+	Prefix   string
 }
 
 func (opts *LoginOptions) AskUserPass() (err error) {
-	opts.Username, err = helper.Ask("Username [%s]: ", config.CurrentUser.Username)
+	opts.Username, err = helper.Ask("%sUsername [%s]: ", opts.Prefix, config.CurrentUser.Username)
```

Comment:
At the moment prefix is used in daemon/install to apply proper indentation to the output.